js/islandora_book_reader.js

```diff
@@ -307,13 +307,24 @@
       }
     }
 
-    var booktitle
-     = this.bookTitle;
+    // Create local vars here, since we can use the 'this' reference in the
+    // each look below.
+    var booktitle = this.settings.label;
+    var number_of_pages = this.settings.pageCount;
+    var pages = this.settings.pages;
     // Add alt tag for screen reader accessibility.
     $('.BRnoselect').each(function(i) {
       // Gives each book page a title, reminiscent of
-      // 'Sample Book: Page 1'.
-      var title = booktitle + ": " + Drupal.t("Page") + " " + (i + 1);
+      // '{Obj label}: {X} of {Y} ({Page obj label})'.
+      var title = Drupal.t(
+        '@label: @x of @y (@pagetitle)',
+        {
+          '@label': booktitle,
+          '@x': i + 1,
```

Comment:
@MorganDawe note here: Just to be sure here: i + 1 could end in 01 , 11, etc. not sure javascript casts it to integer by default. What about (parseInt(i) + 1) or at least (i + 1) ?

Reply:
Sorry for the wait @DiegoPino, this does work as intended. By default in the loop, the iterator is an index seen [here](https://api.jquery.com/each/#each-function) on the jquery docs page. That said i did test it again to be sure. Example markup output below:
```
<img class="BRnoselect" style="width: 510px; height: 660px;" src="http://discoverygarden-vagrant-cmu.dgi/adore-djatoka//resolver?rft_id=http%3A%2F%2Fdiscoverygarden-vagrant-cmu.local%2Fislandora%2Fobject%2Fislandora%253A4%2Fdatastream%2FJP2%2Fview%3Ftoken%3Dbc7dd8988b2dc78d2abb11c6663079711b498f8a4cf107effbfb880b4a7a831f&amp;url_ver=Z39.88-2004&amp;svc_id=info%3Alanl-repo%2Fsvc%2FgetRegion&amp;svc_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajpeg2000&amp;svc.format=image%2Fjpeg&amp;svc.level=4&amp;svc.rotate=0" alt="Test book: 3 of 6 (The Raven.pdf-3)">
```

Notice the ```alt="Test book: 3 of 6 (The Raven.pdf-3)``` part.